src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -192,7 +185,7 @@ private static String getTokenFromRequest(Request req) {
     private static JWTVerifier getVerifier(Request req, String token) {
         if (verifier == null) {
             try {
-                final String domain = "https://" + getConfigPropertyAsText("AUTH0_DOMAIN") + "/";
+                final String domain = "https://" + ConfigUtils.getConfigPropertyAsText("AUTH0_DOMAIN") + "/";
```

Comment:
There are still a lot of changes on this PR related to adding qualifiers for imports from util files. I thought we decided against that?

Reply:
yeah sorry. I think I have now reverted the changes I made previously.